index.html

```diff
@@ -1,12 +1,56 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
-  <meta charset="UTF-8">
-  <meta http-equiv="X-UA-Compatible" content="IE=edge">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Seko Dameh</title>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+    <!-- local resources -->
+    <link rel="stylesheet" href="css/style.css">
+
+    <!-- online resources -->
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap" rel="stylesheet">
+
+    <title>Dameh SekoViper</title>
 </head>
 <body>
-  
+    <header>
+        <nav>
+            <div>
+                <a href="#" class="logo">My Logo</a>
+            </div>
+            <div>
+                <div class="toggler">
+                    <div></div>
+                    <div></div>
+                    <div></div>
+                </div>
+                <div class="menu-items">
+                    <ul>
+                        
+                    </ul>
+                </div>
+            </div>
+        </nav>
+    </header>
+
+    <main>
+        <div class="hero-container">
+            <div class="content-container">
+                <h1>I’m Seko<br> Glad to <br>see you!</h1>
+                <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+                <h4 class="connect">Let's Connect</h4>
```

Comment:
- [ ] The "Let's connect" button looks more like a link to a possibly "contact section" of the page than a heading. An `<a>` tag would be better appropriate for this element. Please change that and remember to use the `href` attribute(you can use `href="#"` for now).